src/components/settings/FeeConfigModal.tsx

```diff
@@ -0,0 +1,562 @@
+import { forwardRef, useRef, useCallback, useEffect, useState } from 'react'
+import * as rb from 'react-bootstrap'
+import { useTranslation } from 'react-i18next'
+import { Formik, FormikErrors } from 'formik'
+import classNames from 'classnames'
+import { useRefreshConfigValues, useUpdateConfigValues } from '../../context/ServiceConfigContext'
+import { factorToPercentage, percentageToFactor } from '../../utils'
+import Sprite from '../Sprite'
+import styles from './FeeConfigModal.module.css'
+import SegmentedTabs from '../SegmentedTabs'
+
+type SatsPerKiloVByte = number
+
+const TX_FEES_BLOCKS_MIN = 1
+const TX_FEES_BLOCKS_MAX = 1_000
+const TX_FEES_SATSPERKILOVBYTE_MIN: SatsPerKiloVByte = 1_000 // 1 sat/vbyte
+// 350 sats/vbyte - no enforcement by JM - this should be a "sane" max value (taken default value of "absurd_fee_per_kb")
+const TX_FEES_SATSPERKILOVBYTE_MAX: SatsPerKiloVByte = 350_000
+const TX_FEES_SATSPERKILOVBYTE_ADJUSTED_MIN = 1_001 // actual min of `tx_fees` if unit is sats/kilo-vbyte
+const TX_FEES_FACTOR_DEFAULT_VAL = 0.2 // 20%
+const TX_FEES_FACTOR_MIN = 0 // 0%
+const TX_FEES_FACTOR_MAX = 1 // 100%
+const CJ_FEE_ABS_MIN = 1
+const CJ_FEE_ABS_MAX = 100_000_000 // 1 BTC - no enforcement by JM - this should be a "sane" max value
```

Comment:
More like insane max value 😅

Reply:
Valid statement. Using 1k BTC would be 0.1% fee - seems high. Don't want to restrict it too much, though. Really don't know an acceptable upper value here. 1mil? 100k? Reduce to 10k even?
Should the current orderbook be taken into account?

Reply:
Lowered to 1mil sats.